apps/admin/frontend/src/api.ts

```diff
@@ -141,10 +145,62 @@ export const unprogramCard = {
   },
 } as const;
 
-export const saveBallotPackageToUsb = {
+// USB
+
+export const getUsbDriveStatus = {
+  queryKey(): QueryKey {
+    return ['getUsbDriveStatus'];
+  },
+  useQuery() {
+    const apiClient = useApiClient();
+    return useQuery(this.queryKey(), () => apiClient.getUsbDriveStatus(), {
+      refetchInterval: USB_DRIVE_STATUS_POLLING_INTERVAL_MS,
```

Comment:
I continue to lament all the polling, but I don't think it makes sense to do differently in this PR.